content/en/docs/tasks/security/webhook/index.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Istio Webhook Management
+description: Shows how to manage webhooks in Istio through istioctl.
```

Comment:
Can you include why a user would want to do this? Adding this doc, which is a lot of complicated and manual steps, makes it look like this is a required thing to user Istio properly

Reply:
Added the following explanation why a user would want to do this:
Istio has two webhooks (Galley and Sidecar Injector). In current Istio implementation,
Galley and Sidecar Injector manage their own webhook configurations, which from
security perspective is not recommended because a compromised webhook may conduct
privilege escalation attacks.

This task shows how to use `istioctl`, instead of Galley and Sidecar Injector, to
manage the webhook configurations of Galley and Sidecar Injector.